Jenkinsfile

```
@@ -353,5 +353,27 @@ void checkCustom() {
             sh "grep -lrE '(NULL|[^_]assert)' . | grep '\\.cpp\\|\\.h'"
             sh "exit 1"
         }
+
+        // check that no file contains an #include <autopas/...>
+        try{
+            // if .cpp or .h files contain #include <autopas, return 2
```

```suggestion
            // if any files contain #include <autopas, return 2
```
We probably should restrict this to source files only